### ISSUE TITLE
Evaluate Mirroring GFOs food project map

### ISSUE THREAD

**Issue by ColmMassey** [SolidarityEconomyAssociation/owned-by-oxford-project#7]
Which fields for the projects on this map https://foodmap.goodfoodoxford.org/ can we model easily? Which don't make sense to model and what is left?

The source data is available on Nextcloud at
..CodeOperatives/Owned by Oxford/Food_GFO_21_10_15.xls

**Comment by wu-lee** [comment]
Fields in the spreadsheet are:
- Organisation Id
- Organisation Created At
- Organisation Updated At
- Organisation Name
- Editor
- Organisation Website
- Organisation Facebook
- Organisation Twitter
- Organisation Spreadsheet Import Id
- Location Id
- Location Created At
- Location Updated At
- Street Address
- City
- Postcode
- Location Telephone
- Location Email
- Message
- Disabled Access
- Mon
- Tue
- Wed
- Thu
- Fri
- Sat
- Sun
- Limited Access
- Access Criteria
- Other Access
- Food Services Provided
- Other Food
- Cost
- Price
- Other Services
- Latitude
- Longitude


Of which I think the following mappings make currently sense:

- id: Organisation Id
- name: Organisation Name
- homepage: Organisation Website
- facebook: Organisation Facebook
- twitter: Organisation Twitter
- street_address: Street Address
- locality: City
- postcode: Postcode
- telephone? Location Telephone
- email? Location Email
- latitude: Latitude
- longitude: Longitude

There are some fields which look like vocabs/enums, which we *could* try to support, but that opens the can-of-worms which is vocabs.

**Comment by ColmMassey** [comment]
>Organisation Id Do you mean Location Id?

>homepage: Organisation Website
facebook: Organisation Facebook
twitter: Organisation Twitter

Are they missing off the first list?

**Comment by ColmMassey** [comment]
How much time required to mirror those fields into a triple store? To be added to OBO set.

**Comment by wu-lee** [comment]
> Are they missing off the first list?

Oops, they were. I've updated the list.  Organisation Id is presumably more correct here... unless the data is one row per location, in which case there may be duplicates. But then there is another problem, which is that the data has more than one location per organisation, which our current `standard.csv` schema doesn't really support.  In which case we might need to fudge things and try to use Location Id.

**Comment by wu-lee** [comment]
> ...To be added to OBO set.

Yes, noticed that in issue #1, which is why I came here.  One of the bigger jobs, I think - 

Having looked at the data more closely, I see that yes, Location Id is unique, Organisation Id is not (Cherwell Larder, for example, has a ton of locations).

This may have implications in the directory panel, such as, should it show lots of duplicate names? And the behaviour will depend on the assumptions written into the code, which I'm not certain about.  I think maybe Coops UK data has a similar structure, with some organisations appearing multiple times?

**Comment by ColmMassey** [comment]
Perhaps we should have a little design brainstorm about options for handling 'outlets'. I think Matt had a design to handle CO-ops UK but it hasn't been reviewed in years. It is no doubt something we will encounter again and again.

**Comment by wu-lee** [comment]
As I mentioned in #1, I've added an open-data conversion for good-food-oxford (still manual currently).

It's using ESSGLOBAL 2.1.  As such I set the Country Id to `GB` and leave Territory Id blank... (I wonder if the latter field really deserves to be in the standard.csv schema, it's very ICA specific.)

I use the Location Id as the prirmary identifier, and merely capture (but ignore) the Organisation Id.

There's no usable description field, unfortunately.

As an aside: Cherwell Larder seems to have used a Google form as its homepage URL... so I think the data could do with some cleaning upstream.
